lib/Jenkins/API.pm

```diff
@@ -547,31 +553,49 @@ sub general_call
 
 =head2 response_code
 
-This method returns the HTTP response code from our last request to 
+This method returns the HTTP response code from our last request to
 the Jenkins server.  This may be useful when an error occurred.
 
 =cut
 
-sub response_code
-{
-    my $self = shift;
-    return $self->_client->responseCode;
-}
-
 =head2 response_content
 
-This method returns the content of the HTTP response from our 
-last request to the Jenkins server.  This may be useful when 
+This method returns the content of the HTTP response from our
+last request to the Jenkins server.  This may be useful when
 an error occurs.
 
 =cut
 
-sub response_content
-{
-    my $self = shift;
-    return $self->_client->responseContent;
-}
+=head2 response_header
+
+This method returns the specified header of the HTTP response from
+our last request to the Jenkins server.  The following example
```

Comment:
I'd suggest starting a new paragraph with "The following example" ?

Reply:
Please feel free to adjust this.